extension/js/common/core/pgp-key.ts

```diff
@@ -36,6 +36,8 @@ export interface PrvKeyInfo {
   parsed?: OpenPGP.key.Key;     // only for internal use in this file
 }
 
+export type KeyType = 'curve25519' | 'rsa2048' | 'rsa4096';
```

Comment:
I would call this `KeyAlgo` for clarity.

Reply:
Done :+1: